qiita_pet/templates/study_description_templates/preprocessed_data_info_tab.html

```diff
@@ -77,7 +77,12 @@ <h4 class="modal-title" id="myModalLabel">Choose processing parameters</h4>
       </tr>
   </table>
   {% if not processed_data or processing_status.startswith('failed') %}
-    <a class="btn btn-primary glyphicon glyphicon-play" data-toggle="modal" data-target="#process-modal-view-{{ppd_id}}" style="word-spacing: -10px;"> Process</a>
+    <a style="word-spacing: -10px;"
+    {% if processing_status == "processing" %}
+      class="btn btn-default glyphicon glyphicon-play" style=" pointer-events: none; cursor: default; background-color:grey;"
```

Comment:
This doesn't seem to disable the button if it is processing. Need to disable it so we don't double-process